src/App.tsx

```diff
@@ -17,18 +54,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onChangeStatusTodos={setTodoStatus}
+                todoStatus={todoStatus}
+                onChangeQuery={setQuery}
+                query={query}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!todos.length
```

Comment:
Use shortcuts for booleans, but explicit comparisons for strings and numbers:
https://mate-academy.github.io/style-guides/javascript.html#153use-shortcuts-for-booleans-but-explicit-comparisons-for-strings-and-numbers
```suggestion
              {todos.length === 0
```